salt/modules/boto_dynamodb.py

```diff
@@ -233,7 +265,7 @@ def describe(table_name, region=None, key=None, keyid=None, profile=None):
     return table.describe()
 
 
-def _extract_index(index_data, global_index=False):
+def extract_index(index_data, global_index=False):
```

Comment:
This is very specific to dynamodb, but it's needed from both within and outside of the module. It would be nice to keep this inside of the dynamodb module if possible. Worst case we can just add a basic cli example and mention it's not really intended for CLI use.